src/manifests/bundle_manifest.py

```diff
@@ -89,10 +89,25 @@ def get_tarball_relative_location(build_id, opensearch_version, platform, archit
         # TODO: use platform, https://github.com/opensearch-project/opensearch-build/issues/669
         return f"bundles/{opensearch_version}/{build_id}/{architecture}/opensearch-{opensearch_version}-{platform}-{architecture}.tar.gz"
 
+    @staticmethod
+    def get_tarball_relative_location_for_dashboards(build_id, opensearch_version, platform, architecture):
+        # return f"bundles/{opensearch_version}/{build_id}/{architecture}/opensearch-{opensearch_version}-{platform}-{architecture}.tar.gz"
+        # The path above has no 1.2.0 bundle. Thus use the new path and a hardcoded build ID for now.
+        # https://github.com/opensearch-project/opensearch-build/issues/838
+        return f"bundle-build-dashboards/{opensearch_version}/195/{platform}/{architecture}/dist/opensearch-dashboards-{opensearch_version}-{platform}-{architecture}.tar.gz"
```

Comment:
any reason to hard code build id?

Reply:
Dashboards and OpenSearch have different build IDs. The execution command to run integ test has not yet taken OSD build ID as input yet. (will try to address this in https://github.com/opensearch-project/opensearch-build/issues/838) Thus for now, to minimize the change, I will hardcode a build ID for OSD.